openedx/core/djangoapps/credit/migrations/0008_auto__del_index_creditprovider_provider_id__chg_field_crediteligibilit.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Changing field 'CreditEligibility.provider'
+        db.alter_column('credit_crediteligibility', 'provider_id', self.gf('django.db.models.fields.related.ForeignKey')(null=True, to=orm['credit.CreditProvider']))
+
+    def backwards(self, orm):
+        # Changing field 'CreditEligibility.provider'
+        db.alter_column('credit_crediteligibility', 'provider_id', self.gf('django.db.models.fields.related.ForeignKey')(default=1, to=orm['credit.CreditProvider']))
```

Comment:
@feanil This migration is first making the field nullable (rather than setting the default to 1), then deleting it.  We could separate this into two PRs, but the code that uses this isn't enabled in production, so I'd prefer to do this in one release if possible.  Does that seem reasonable?